inst/tests/tests.Rraw

```diff
@@ -15941,11 +15941,17 @@ test(2067.8, shift(list(z, 1L:3L), n=-1, type = 'cyclic'), list(c(z[2:3], z[1]),
 DT = data.table(a = 2:1, z = complex(0, 0:1))
 test(2068.1, setkey(copy(DT), a), data.table(a=1:2, z=complex(0, 1:0), key='a'))
 test(2068.2, DT[ , abs(z), by=a], data.table(a=2:1, V1=c(0, 1)))
-# raw continues not to be supported
+# support for ordering tables with raw columns, #5100
 DT = data.table(ID=2:1, r=as.raw(0:1))
-test(2068.3, setkey(DT, ID), error="Item 2 of list is type 'raw'")
```

Comment:
keep an eye on codecov here -- one of the reasons I hesitate a bit on #5100 is I used raw several times as the go-to "coverage test for unsupported type". which isn't a good reason not to implement features, but means extra overhead of rewriting coverage tests

Reply:
I saw and acknowledged that but also used `expression`s as replacement for raw to not drop coverage (see test 2068.6).

Maybe there is even a better replacement than `expression` when looking at [R.internals.h](https://github.com/wch/r-source/blob/0cda2520430ba3ffb80073503bbd2651c4bb8866/src/include/Rinternals.h#L143-L174)